src/handlers/payout/post.ts

```diff
@@ -21,26 +24,63 @@ export const incentivizeComments = async () => {
   }
   const context = getBotContext();
   const payload = context.payload as Payload;
-  const org = payload.organization?.login;
   const issue = payload.issue;
-  if (!issue || !org) {
-    logger.info(`Incomplete payload. issue: ${issue}, org: ${org}`);
+  if (!issue) {
+    logger.info(`Incomplete payload. issue: ${issue}`);
     return;
   }
+
+  if (issue.state_reason !== StateReason.COMPLETED) {
+    logger.info("incentivizeComments: comment incentives skipped because the issue was not closed as completed");
+    return;
+  }
+
+  if (paymentPermitMaxPrice == 0 || !paymentPermitMaxPrice) {
```

Comment:
`paymentPermitMaxPrice` is just being checked here for having some value, but it is not being enforced in this function. A big incentive is being generated https://github.com/web4erOrg/ubiquibot/issues/56